src/components/AddressSearch.js

```diff
@@ -184,7 +184,7 @@ const AddressSearch = (props) => {
                     query={query}
                     requestUrl={{
                         useOnPlatform: 'all',
-                        url: `${CONFIG.EXPENSIFY.URL_API_ROOT}api?command=Proxy_GooglePlaces&proxyUrl=`,
+                        url: `${CONFIG.EXPENSIFY.PRIMARY_API_ROOT}api?command=Proxy_GooglePlaces&proxyUrl=`,
```

Comment:
Should this use `ApiUtils.getApiRoot()`?

Reply:
I had the same question, I guess I forgot to post it 

I think since we're letting the "staging toggle" change the API for attachments we should follow the same here

Reply:
Agree